library.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+from library.model.book import Book
+from library.model.genre import Genre
+
+
+class Library():
+
+    # Prints crime audio books
+    def print_longest_crime_audio_book(self) -> str:
+        crime_audio_books: list[Book] = []
+        max: Optional[Book] = None
```

Comment:
The `max` variable does not really tell us what this variable should be holding. Renaming it to `longest_book` would make it clearer
```suggestion
        longest_book: Optional[Book] = None 
```